src/Migrations/Version20250107182806.php

```diff
@@ -0,0 +1,31 @@
+<?php
```

Comment:
technically this migration should be deployed after the other change, but in practice it doesn't matter if its a few seconds earlier.